src/index.njk

```diff
@@ -5,11 +5,11 @@ layout: base.njk
 {% from 'macros/cards.njk' import feedCard %}
 {% include 'partials/banner.njk' %}
 <main>
-  <div class="mx-auto w-full bg-gray-300 items-center">
-    <div class="py-10 px-5 md:px-20 bg-gray-300 dark:bg-gray-800">
+  <div class="mx-auto w-full items-center">
+    <div class="py-10 px-5 md:px-20 bg-ivory dark:bg-darkGray">
        <div class="mx-auto max-w-screen-xl grid sm:grid-cols-1 md:grid-cols-1 lg:grid-cols-3 grid-rows-1 gap-4 justify-items-center">
       {% asyncEach feed in collections.feeds %}
-        <section class="w-full py-4 px-4 m-2 shadow-md dark:shadow-md-light h-full bg-white dark:bg-gray-900 rounded-md">
+        <section class="w-full py-4 px-4 m-2 shadow-md h-full bg-ivory dark:bg-cardDarkGray rounded-md hover:shadow-cardShadow">
```

Comment:
Please add appropriate shadow classes for dark mode as well.

Reply:
added border instead